<launch>

  <!-- camera -->
  <!--
  <node pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera_node"/>
  -->

  <!-- qr reader -->
  <!--
  <node pkg="onigiri_war" type="qrReader.py" name="qr_reader" output="screen"/>
  -->

  <!-- send to judge server -->
  <node pkg="onigiri_war" type="sendQrToJudge.py" name="send_qr_to_judge" output="screen">
    <param name="judge_url" value="http://127.0.0.1:5000/submits" />
    <param name="player_name" value="roborobo" />
    <param name="side" value="r" />
  </node>

  <!-- robot controller -->
  <!--
  <include file="$(find navi_param)/rulo_navi.launch" />
  -->

</launch>
